USAGE_desktop.md

```diff
@@ -119,66 +123,12 @@ export VK_INSTANCE_LAYERS=VK_LAYER_LUNARG_gfxreconstruct
 ### Capture Options
 
 The GFXReconstruct layer supports several options, which may be enabled
-through environment variables or a layer settings file.
-
-#### Windows Options
```

Comment:
Is it important to remove these two hints to users about how to set environment variables on Windows and Linux?

Reply:
I'm trying to avoid mentioning specific options in layer documentation, in case a change is made to a layer that obsoletes some option. We'll let the automatically generated documentation document the options. To that end, I tried to remove options mentioned in layer overview documents, although I realized there is still some options mentioned in this document, which to remove, would almost require a complete rewrite of the gfxreconstruct documentation.

Reply:
Put this back in.